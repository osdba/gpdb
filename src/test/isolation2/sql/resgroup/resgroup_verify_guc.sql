-- start_ignore
! gpconfig -c max_statement_mem -v 20GB
! gpconfig -c statement_mem -v 10GB
! gpstop -rai;
-- end_ignore

show max_statement_mem;
show statement_mem;
