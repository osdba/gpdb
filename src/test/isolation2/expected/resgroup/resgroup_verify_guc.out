-- start_ignore
! gpconfig -c max_statement_mem -v 20GB ! gpconfig -c statement_mem -v 10GB ! gpstop -rai;
20170929:03:24:11:381552 gpconfig:ad3e397dc7b9:gpadmin-[INFO]:-completed successfully with parameters '-c max_statement_mem -v 20GB'
20170929:03:24:23:381640 gpconfig:ad3e397dc7b9:gpadmin-[INFO]:-completed successfully with parameters '-c statement_mem -v 10GB'
20170929:03:24:23:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Starting gpstop with args: -rai
20170929:03:24:23:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Gathering information and validating the environment...
20170929:03:24:23:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Obtaining Greenplum Master catalog information
20170929:03:24:23:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Obtaining Segment details from master...
20170929:03:24:25:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Greenplum Version: 'postgres (Greenplum Database) 5.0.0-beta.10+dev.814.g6e61528 build dev'
20170929:03:24:25:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-There are 0 connections to the database
20170929:03:24:25:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Commencing Master instance shutdown with mode='immediate'
20170929:03:24:25:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Master host=ad3e397dc7b9
20170929:03:24:25:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Commencing Master instance shutdown with mode=immediate
20170929:03:24:25:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Master segment instance directory=/home/gpadmin/workspace/gpdb/gpAux/gpdemo/datadirs/qddir/demoDataDir-1
20170929:03:24:26:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Attempting forceful termination of any leftover master process
20170929:03:24:27:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Terminating processes for segment /home/gpadmin/workspace/gpdb/gpAux/gpdemo/datadirs/qddir/demoDataDir-1
20170929:03:24:28:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-No standby master host configured
20170929:03:24:28:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Commencing parallel primary segment instance shutdown, please wait...
20170929:03:24:28:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-0.00% of jobs completed
20170929:03:24:38:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-100.00% of jobs completed
20170929:03:24:38:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Commencing parallel mirror segment instance shutdown, please wait...
20170929:03:24:38:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-0.00% of jobs completed
20170929:03:24:48:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-100.00% of jobs completed
20170929:03:24:48:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-----------------------------------------------------
20170929:03:24:48:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-   Segments stopped successfully      = 6
20170929:03:24:48:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-   Segments with errors during stop   = 0
20170929:03:24:48:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-----------------------------------------------------
20170929:03:24:48:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Successfully shutdown 6 of 6 segment instances 
20170929:03:24:48:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Database successfully shutdown with no errors reported
20170929:03:24:48:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Cleaning up leftover gpmmon process
20170929:03:24:49:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-No leftover gpmmon process found
20170929:03:24:49:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Cleaning up leftover gpsmon processes
20170929:03:24:50:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-No leftover gpsmon processes on some hosts. not attempting forceful termination on these hosts
20170929:03:24:50:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Cleaning up leftover shared memory
20170929:03:24:51:381729 gpstop:ad3e397dc7b9:gpadmin-[INFO]:-Restarting System...

-- end_ignore

show max_statement_mem;
max_statement_mem
-----------------
20GB             
(1 row)
show statement_mem;
statement_mem
-------------
10GB         
(1 row)
